demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -76,7 +76,7 @@ def prepare_hooks(pre_commit_context: PreCommitContext) -> None:
             "sourcery": SourceryHook,
             "validate": ValidateFormatHook,
             "format": ValidateFormatHook,
-            "mypy": MypyHook,
+            "mypy": MypyHook,  # TODO delete this hook after https://github.com/demisto/content/pull/35880 will merged
```

Comment:
```suggestion
            "mypy": MypyHook,
```
Please open an issue for this and assign yourself, no need to add it in code.

Reply:
https://jira-dc.paloaltonetworks.com/browse/CIAC-11830